public/canvas/index.js

```diff
@@ -0,0 +1,33 @@
+(function(window, $) {
+  'use strict';
+
+  var loadCanvasCustomization = function() {
+    window.CALCENTRAL = 'https://junction.berkeley.edu';
+
+    // Ensure the bCourses development and test servers are pointing to the correct
+    // CalCentral instance when a copy of production is made
+    if (window.location.origin === 'https://ucberkeley.beta.instructure.com') {
+      window.CALCENTRAL = 'https://junction-dev.berkeley.edu';
+    } else if (window.location.origin === 'https://ucberkeley.test.instructure.com') {
+      window.CALCENTRAL = 'https://junction-qa.berkeley.edu';
+    }
+
+    // Load the JavaScript customizations
+    $.getScript(window.CALCENTRAL + '/static/canvas/canvas-customization.js');
+
+    // Load the CSS customizations
+    var css = $('<link>', {
+      'rel': 'stylesheet',
+      'type': 'text/css',
+      'href': window.CALCENTRAL + '/static/canvas/canvas-customization.css'
```

Comment:
@mlincoln - I think my PR review is done. Keep an eye out for any use of `window.CALCENTRAL` (as seen above).  Those URLs will likely need to be modified to the Ripley way.

Reply:
I missed this one. Will amend again. 

Waiting to see if my .gitignore change addressed the travis failures.